easybuild/tools/modules.py

```diff
@@ -884,24 +875,22 @@ def get_software_root(name, with_env_var=False):
     """
     Return the software root set for a particular software name.
     """
-    environment_key = get_software_root_env_var_name(name)
-    newname = convert_name(name, upper=True)
-    legacy_key = "SOFTROOT%s" % newname
+    env_var = get_software_root_env_var_name(name)
+    legacy_key = "SOFTROOT%s" % convert_name(name, upper=True)
```

Comment:
the intention is to remove all `nosupport(<msg>, '2.0')` calls in `3.0`, so adding an extra `log.deprecated` doesn't really help :)

it'll fire an error already anyway, so an extra deprecation warning on top is pretty useless indeed